components/TripList/index.js

```diff
@@ -10,15 +11,19 @@ export default function TripList() {
     <ul>
       {trips.map((trip) => {
         return (
-          <Trip
-            key={trip._id}
-            image={trip.image}
-            title={trip.title}
-            startDate={trip.startDate}
-            endDate={trip.endDate}
-            city={trip.city}
-            country={trip.country}
-          />
+          <>
+            <Link href={`/trips/${trip._id}`}>
+              <Trip
+                key={trip._id}
+                image={trip.image}
+                title={trip.title}
+                startDate={trip.startDate}
+                endDate={trip.endDate}
+                city={trip.city}
+                country={trip.country}
+              />
+            </Link>
+          </>
         );
       })}
     </ul>
```

Comment:
Your HTML structure here looks something like:

```html
<ul>
  <a href="..."><li>...</li></a>
  <a href="..."><li>...</li></a>
...
</ul>
```

However, the direct children of a `ul` would be the list items (`li`). Therefore I suggest to include the `Link` component in your `Trip` component.

So, your `TripList` would look like this:

```jsx
 <ul>
      {trips.map((trip) => {
        return (
         
              <Trip
                key={trip._id}
                id={trip._id}
                image={trip.image}
                title={trip.title}
                startDate={trip.startDate}
                endDate={trip.endDate}
                city={trip.city}
                country={trip.country}
              />
        );
      })}
    </ul>
```

And your `Trip` component:

```jsx
 <li>
  <Link href={`/trips/${trip._id}`}>
...
  </Link>
 </li>

```